includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -31,6 +31,36 @@ public function getNavigation() {
 		$this->addCachingHeaders();
 	}
 
+	public function getCommunityHeader() {
+		$params = $this->getRequestParameters();
+
+		//TODO: think about refactoring ThemeSettings class to have cityId as constructor param
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)